src/main/java/com/sonyericsson/rebuild/RebuildAction.java

```diff
@@ -226,12 +226,11 @@ public void parameterizedRebuild(Run currentBuild, StaplerResponse response) thr
         if (project == null) {
             return;
         }
+        System.out.println("rbx: " + (currentBuild == build));
```

Comment:
Please avoid System.out. Use logging if text output necessary (but it is?)

Reply:
It's my debugging output. Been removed.